account_bank_statement_import_camt_oca/models/parser.py

```diff
@@ -199,6 +200,17 @@ def parse_statement(self, ns, node):
                 './ns:Acct/ns:Id/ns:Othr/ns:Id',
             ], result, 'account_number'
         )
+
+        if result["account_number"]:
+            # find the journal related to this account
+            journals = self.env["account.journal"].search([])
+            trans_table = str.maketrans("", "", string.whitespace)
+            account_number_ = result["account_number"].translate(trans_table)
+            journal = journals.filtered(
+                lambda x: x.bank_acc_number and x.bank_acc_number.translate(trans_table) == account_number_)
```

Comment:
bank_acc_number is a required field so it is always set in theory.
```suggestion
                lambda x: x.bank_acc_number.translate(trans_table) == account_number_)
```

Reply:
In my database it's not set for every record

![image](https://user-images.githubusercontent.com/15734398/150930025-aee7aed5-c0f6-46fa-8e8f-9d7a06d1a2cb.png)